tests/unittests/OperatorTest.cpp

```diff
@@ -5063,6 +5063,94 @@ TEST_P(InterpOnly, Modulo2) {
   }
 }
 
+/// Test a DotProduct operator with 1D inputs.
```

Comment:
it's not for this PR, but this test is huge right now, we might need to split it to speed up execution of tests.